runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -2337,6 +2336,17 @@ MM_CopyForwardScheme::scanContinuationObjectSlots(MM_EnvironmentVLHGC *env, MM_A
 	}
 }
 
+MMINLINE void
+MM_CopyForwardScheme::scanContinuationObjectSlots(MM_EnvironmentVLHGC *env, MM_AllocationContextTarok *reservingContext, J9Object *objectPtr, ScanReason reason)
+{
+	scanContinuationNativeSlots(env, reservingContext, objectPtr, reason);
+	scanMixedObjectSlots(env, reservingContext, objectPtr, reason);
+}
+
+MMINLINE void
+MM_CopyForwardScheme::scanContinuationNativeSlots(MM_EnvironmentVLHGC *env, MM_AllocationContextTarok *reservingContext, J9Object *objectPtr, ScanReason reason)
```

Comment:
this is a duplicate definition (got one already from renaming scanContinuationObjectSlots)